kingpin/actors/rightscale/base.py

```diff
@@ -195,3 +197,18 @@ def flatten(d, parent_key=prefix, sep='_'):
             return items
 
         return flatten(params)
+
+    @gen.coroutine
+    def _log_account_name(self, *args, **kwargs):
+        if not RightScaleBaseActor.account_name:
+            ca_resource = self._client._client.cloud_accounts
+            cloud_accounts = yield self._client.show(ca_resource)
+            account = yield self._client.show(cloud_accounts[0].account)
+            RightScaleBaseActor.account_name = account.soul['name']
+            log.warn('RightScale account name: %s' % account.soul['name'])
+
+    @gen.coroutine
+    def execute(self, *args, **kwargs):
+        self._log_account_name()
```

Comment:
Mocking it out in every test shouldn't be too hard -- just mock it out in the `setUp()` method of the test suite?